docs/config/README.md

```diff
@@ -1399,6 +1399,10 @@ The module will be shown only if there are background jobs running.
 The module will show the number of jobs running if there is more than 1 job, or
 more than the `threshold` config value, if it exists.
 
+::: warning
+This module is not supported on tcsh.
+:::
```

Comment:
I can't remember the exact reasons but I think we need to have additional line breaks around the `:::` to work better with CrowdIn.
```suggestion
::: warning

This module is not supported on tcsh.

:::
```